Source/Models.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  Models.swift
```

Comment:
Totally might be jumping the gun, but any specific reason to keep these classes grouped versus in their own respective swift files?